### PR TITLE
Adds a new brain damage symptom simply called Brain Degradation (Almost Ready)

### PR DESCRIPTION
**Pull request by davethwave**
Adds a new brain damage symptom that has the same stats as the recently removed Viral Aggressive Metabolism. Due to me being new to byond coding there is nothing really special except that the virus does brain damage over time. As this is my first PR please judge freely.

:cl: davethwave
add: Added a new symptom that does brain damage
/:cl:

I made this to replace the stats missing from the Viral Aggressive Metabolism as well as add something that I though was fun to the game.

### CODE REVIEW

**Comment by arsserpentarium** [comment]
homemade brainrot

**Comment by davethwave** [comment]
If this gets test-merged and people think it's too OP then I will nerf it.

**Comment by davethwave** [comment]
Hope they fix trauma healing via mannitol before this is test-merged or merged or this will be interesting. Because as it is right now you can only heal trauma's via surgery.

File: code/datums/diseases/advance/symptoms/migraine
Comment (PKPenguin321) [inline code/datums/diseases/advance/symptoms/migraine]: "...Impedrezene, **which** effectively removes..."

Comment (davethwave) [reply code/datums/diseases/advance/symptoms/migraine]: fixed that

**Comment by davethwave** [comment]
I'll also see if it's possible for me after the freeze to add some stat based effects if I can get a better understanding of byond code.

**Comment by davethwave** [comment]
4 days until the freeze...

**Comment by imsxz** [comment]
The confusion symptom causes brain damage as well when certain thresholds are met, may be something to consider

**Comment by davethwave** [comment]
Never knew that

File: code/datums/diseases/advance/symptoms/migraine
Comment (ShizCalev) [inline code/datums/diseases/advance/symptoms/migraine]: trailing return

Comment (davethwave) [reply code/datums/diseases/advance/symptoms/migraine]: So take off the 'return' at the end?

**Comment by davethwave** [comment]
I decided to add a damage threshold to how much  brain damage you can accumulate with the symptom as well as some stat stuff so I'll put WIP on this for now. Will probably take a few days due to my incompetence and inexperience with byond code.

**Comment by KorPhaeron** [comment]
Ping me if you're going to finish this, 7 days without an update though

**Comment by davethwave** [comment]
Working on it, just new to byond.

**Comment by davethwave** [comment]
I honestly can't seem to find out how to add a brain damage threshold and the like so for now I'll put the virus symptom to level 7 and buff the damage a bit and call it good if that's ok.

**Comment by davethwave** [comment]
This is ready as it is right now.

**Review by ninjanomnom — CHANGES_REQUESTED** [review]
You may want to test and see if it even works (hint: it doesn't)

You need to rename the file to `migraine.dm` and include it in the dme

**Comment by optimumtact** [comment]
no maintainer, no change, no life

**Pull request closed without merging**
